### ISSUE TITLE
Add Limitation No IE11 To Overmindjs Home

### ISSUE THREAD

**Issue by ElliotPsyIT** [cerebral/overmind#186]
Very appealing state management toolset!  

I spent some time reviewing the Overmindjs site, but it was not until I read the linked to intro article did I see this library depends on js proxy -- not available in IE.  

It might be nice for this browser limitation to be indicated on the home page as a heads up to web developers who require IE11 support for their state management tools. Perhaps add a note, or maybe return info from a 'Search ...' for IE11 support on the navbar. 

Thanks!

**Comment by shavyg2** [comment]
I think typescript can support proxies with es5
https://github.com/cerebral/overmind/blob/next/packages/node_modules/overmind/tsconfig.json

Would need to confirm but is that still possible.

I think it's using getters and setters which is what i believe is being used?

**Comment by ElliotPsyIT** [comment]
If the Typescript target is ES5 it won't load the necessary Proxy support definitions. See [this StackOverflow discussion](https://stackoverflow.com/questions/46397398/cannot-import-typings-for-proxy) on the topic.

**Comment by christianalfoni** [comment]
Yeah, sorry, not possible to polyfill what we need proxies for. There is talk about introducing an API to support IE11. But I am going to write an article first about thinking differently about support IE11. Cause it has a huge cost on everyone. 

Added note on this, will make sure to point it out in docs :)

**Comment by ElliotPsyIT** [comment]
Thanks - I look forward to your thoughts on IE11 support, @christianalfoni. 

Unfortunately, our large health care system (US Federal Govt - Veteran Affairs) is a enterprise Windows 7 environment - so providing IE11 support is a must for now.  There is a transition to Windows 10 underway, which will set Edge as the default, but across 140 hospitals this effort will take a while :)

I appreciate you adding the IE11 limitation to the docs. Very good of you!

**Comment by christianalfoni** [comment]
Will be part of next release, this evening :)

**Comment by trickydisco78** [comment]
I've been reading the overmind stuff with interested and was also gutted when i realised it won't support IE11. We have to support a lot of products on IE11..

**Comment by chimon2000** [comment]
What about using [immer](https://github.com/mweststrate/immer) for mutations instead of proxies?

**Comment by skurpi** [comment]
First off, Thank you for a great product.

Regarding this issue with IE11: where on the website/docs is this information? We just spent a few months building an app, and at the end of it we started testing IE11 only to find out it does not work. Obviously, WE screwed up greatly and it is fully on us. But a having this information in the docs can be beneficial for people in the future looking into overmind.
